plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployCommandHandler.java

```diff
@@ -69,6 +80,18 @@ public void done(IJobChangeEvent event) {
     deploy.schedule();
   }
 
+  private IPath createWorkDirectory() throws IOException {
+    String now = Long.toString(System.currentTimeMillis());
+    IPath workDirectory = getTempDir().append(now);
+    Files.createDirectories(workDirectory.toFile().toPath(), new FileAttribute<?>[0]);
```

Comment:
You're right, I've missed this. Fixed here and at the other locations in the PR.